make/test.mk

```
@@ -350,6 +350,12 @@ ifneq ($(E2E_TEST_EXECUTION),true)
 	oc delete pods --namespace ${HOST_NS} -l name=registration-service || true
 endif
 
+.PHONY: create-spaceprovisionerconfigs-for-members
+create-spaceprovisionerconfigs-for-members:
+	for member_name in `oc get toolchaincluster -l type=member -n ${HOST_NS} --no-headers -o custom-columns=":metadata.name"`; do \
```

it's just a detail, but we want to drop the type label, so you could do it here right away.
```suggestion
	for member_name in `oc get toolchaincluster -l -n ${HOST_NS} --no-headers -o custom-columns=":metadata.name"`; do \
```
@fbm3307 please don't forget to check also the makefile targets for any occurrences of the usage of the `type` label